src/modules/75-cet/pages/__test__/CETMonitoredServices.test.tsx

```diff
@@ -25,18 +25,18 @@ export const testWrapperProps: TestWrapperProps = {
 const WrapperComponent = (): React.ReactElement => {
   return (
     <TestWrapper {...testWrapperProps}>
-      <ETMonitoredServices />
+      <CETMonitoredServices />
     </TestWrapper>
   )
 }
 
-describe('Unit tests for ETMonitoredServices', () => {
-  test('Verify ETMonitoredServices page renders and matches snapshot', async () => {
+describe('Unit tests for CETMonitoredServices', () => {
+  test('Verify CETMonitoredServices page renders and matches snapshot', async () => {
     const container = render(<WrapperComponent />)
     expect(container).toMatchSnapshot()
```

Comment:
Just FYI, this doesn't get you the rendered output, it just gets you the object. If you want the rendered output, you'll need to get the `container` element from the return. As is this test doesn't prove anything since it's just a snapshot of an object.
```suggestion
    const { container } = render(<WrapperComponent />)
    expect(container).toMatchSnapshot()
```